README.md

```diff
@@ -95,4 +122,4 @@ Then anyone could run that pushed image and have a look if everything is alright
 docker run -p 5000:5000 -it mesosphere/archived_docs:next
 ```
 
-If you merge your branch to the `archive`-branch, jenkins will take care of building and pushing the image.
+If you merge your branch to the `archive`-branch, CI will take care of building and pushing the image.
```

Comment:
this whole section needs to be revised and removed. I think we'll just want to tag `main` to archive going forward and build a docker image from it. TBD.